masterlist.yaml

```diff
@@ -8170,6 +8170,33 @@ plugins:
     url: [ 'https://www.nexusmods.com/oblivion/mods/26570/' ]
     group: *mapGroup
 
+  - name: 'Map Marker Overhaul( - SI additions)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/26389/'
+        name: 'Map Marker Overhaul'
+  - name: 'Map Marker Overhaul.esp'
+    inc:
+      - 'Landmarks, w Wells.esp'
+      - 'Oblivion Gate Map Marker Remover.esp'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Thieves Arsenal'
+          - '[Thieves Arsenal](https://www.nexusmods.com/oblivion/mods/9655/)'
+        condition: 'active("Thieves Arsenal.esp") and not file("Thieves_Arsenal_Patch_033010.esp")'
```

Comment:
`*patch3rdParty` implies the patch is hosted on a page other than that of either mod patched, so an equivalent `*patchProvided` message in _Thieves Arsenal.esp_'s entry would be more appropriate.

```suggestion
```